orchestrator-salt/src/main/resources/salt/salt/gateway/config/cm/topology_token.xml.j2

```diff
@@ -147,6 +147,22 @@
                  <name>{{ aclsauthz_prefix ~ "ssb-sse-api.acl" }}</name>
                  <value>*;{{ salt['pillar.get']('gateway:envAccessGroup') }};*</value>
             </param>
+	    <param>
```

Comment:
Shouldn't this be behind a condition like in other salt related files of this PR?

Reply:
can we merge this or is this still necessary? @biharitomi